### PR TITLE
Fix css being shown as text in reply

### LINKED ISSUE #3863: CSS is shown as txt reply
### Expected behavior
When clicking reply to a message, css is then imported into the reply as text on screen.  This shouldn't be.  You do not see any of this when forwarding the message.

### Actual behavior
When clicking reply to a message, css is then imported into the reply as text on screen.

### Mail app
**Mail app version:1.5.0

### Server configuration
**Operating system: Debain 10
**Web server: Nginx
**Database: Mariadb
**PHP version: 7.0
**Nextcloud Version: 20.0.0

Screen Shot 2020-10-21 at 1 25 58 PM

### PR DESCRIPTION
**Pull request by st3iny**
Fixes #3863 

Ckeditor is converting stylesheets to plain text elements. Strip all stylesheets from the message html prior to passing them to ckeditor.

### CODE REVIEW

File: src/util/sanitize.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param {string} html html
+ * @returns {string}
+ */
+export const stripStylesheets = (html) => {
+	const el = document.createElement('div')
+	el.innerHTML = html
```
Comment (ChristophWurst) [inline src/util/sanitize.js:28]: :warning: aren't you hereby loading potentially dangerous HTML in the DOM?

Comment (st3iny) [reply src/util/sanitize.js:28]: The html has already been sanitized by the backend but this might still be dangerous. DOMPurify should be used here.

Comment (st3iny) [reply src/util/sanitize.js]: Replaced my custom code with DOMPurify.

**Review by ChristophWurst — APPROVED** [review]

**Comment by st3iny** [comment]
Should I rebase to trigger the new test pipeline?

**Comment by ChristophWurst** [comment]
> Should I rebase to trigger the new test pipeline?

Yes please :)

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**